### PR TITLE
Add 'Start Maintenance' action to Host

### PR DESCRIPTION
**Pull request by jtomasek**
~~Depends on https://github.com/kubevirt/web-ui/pull/274~~
Depends on https://github.com/kubevirt/web-ui-components/pull/355

### CODE REVIEW

File: frontend/public/metalkube/components/host/host.tsx [hunk 1]
```diff
@@ -56,12 +58,11 @@ const HostHeader = props => (
 const HostRow = ({ obj: host }) => {
   const {
     metadata: { name, namespace, uid },
```
Comment (suomiy) [inline frontend/public/metalkube/components/host/host.tsx]: you can also use selectors for these.

File: frontend/public/metalkube/components/host/menu-actions.ts [hunk 2]
```diff
@@ -1,25 +1,55 @@
...
+    callback: () => startMaintenanceModal({ resource: node }),
+  };
+};
+
+const menuActionStopMaintenance = (kind, host, { machine, Node: nodes }) => {
+  const node = getMachineNode(nodes, machine);
+  return {
+    hidden: !canHostStopMaintenance(node),
+    label: 'Stop Maintenance',
+    callback: () => makeNodeSchedulable(node),
```
Comment (suomiy) [inline frontend/public/metalkube/components/host/menu-actions.ts]: shouldn't there be a yes/no dialog?

Comment (jtomasek) [reply frontend/public/metalkube/components/host/menu-actions.ts]: Done

File: frontend/public/metalkube/components/host/host.tsx (hunk 1 above)
Comment (jtomasek) [reply frontend/public/metalkube/components/host/host.tsx]: Done

**Pull request merged**